src/AjaxUploader.tsx

```
@@ -305,6 +305,11 @@ class AjaxUploader extends Component<UploadProps> {
     this.fileInput = node;
   };
 
+  eventHandler = (fn: (() => boolean) | boolean, originFn: Function, e: React.MouseEvent<HTMLDivElement> | React.KeyboardEvent<HTMLDivElement>) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)